packages/theme/components/layout/index-cards/IndexCards.tsx

```diff
@@ -31,7 +30,7 @@ export function IndexCards({route, folderData}: IndexCardsProps) {
         if (item.kind !== 'MdxPage') return null
         return (
           <Grid.Column span={{medium: 6}} key={`cell-${item.route}`}>
-            <Card href={`${basePath}${item.route}`} hasBorder style={{width: '100%'}}>
```

Comment:
@rezrah Actually, I think `basePath` should stay here?

Reply:
Yeah, you'll still need it there. Or you could wrap it in a Next <Link>.